test/Main.purs

```diff
@@ -29,7 +28,7 @@ import Web.DOM.Element                   (Element, fromNode, getAttribute)
 import Web.DOM.Node                      (Node, nodeName)
 
 unsafeFromRight :: forall l r. Either l r -> r
-unsafeFromRight = either (unsafeCrashWith "Value was not Left") identity
+unsafeFromRight = fromRight' (\_ -> unsafeCrashWith "Value was not Right")
```

Comment:
Whoops!

Reply:
@JordanMartinez hmm, good catch - if you're referring to the "Left" in the message, I think that was probably a typo on my part, but it should have read "Value was not RIght", as it does now.

Reply:
@bbarker Not quite. "Value was not Left" would only appear if the value was Left, so it doesn't make sense. "Value was not Right" would only appear if the value was Left, not right. Thus, this message is more accurate. I was saying 'whoops' in regards to what I originally wrote here because it's not correct.